libwget/ssl_wolfssl.c

```diff
@@ -594,10 +594,9 @@ void wget_ssl_init(void)
 
 		debug_printf("WolfSSL init\n");
 		wolfSSL_Init();
+#ifndef  NO_OLD_TLS
 
-		if (!wget_strcasecmp_ascii(config.secure_protocol, "SSLv2")) {
-			method = SSLv2_client_method();
-		} else if (!wget_strcasecmp_ascii(config.secure_protocol, "SSLv3")) {
+		if (!wget_strcasecmp_ascii(config.secure_protocol, "SSLv3") || !wget_strcasecmp_ascii(config.secure_protocol, "SSL")) {
```

Comment:
```suggestion
		if (!wget_strcasencmp_ascii(config.secure_protocol, "SSL", 3)) {
```

Reply:
I can make this change, only issue is if someone specifies SSLv2 it won't tell them invalid option but they also won't get what they ask for.  I assumed we would prefer fall to the default case of `Missing TLS method`.  Confirm this is desired instead?

Reply:
You raise a good point. I just compared the WolfSSL code with our OpenSSL and GnuTLS implementations, that why I suggested the above. IMO, all three should behave semantically the same. But I now realize that the docs need to be changed and maybe in a follow-up PR, we need a bit of code deduplication.

So yes, please change the code as suggested above.

I take care of the docs later, and print a warning that SSLv3 is used instead of `SSLv2` (if explicitly requested).